Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -152,9 +157,20 @@ def setupParallelPipelines(
                 }
             }
         } catch(err) {
-            error (errorMessage(machineName, err.message))
+            if (err.message != null) {
+                result[machineName] = err.message
+            }
         }
     }
 }
 
 parallel clones
+
+println "About to print error messages"
+String errors = "\n" \\ This is so that the first line after "Error" in the console
```

Comment:
`\\` is not the syntax for a comment in groovy

Reply:
ah, my apologies, fixed it!